gradle/versions.gradle

```diff
@@ -5,18 +5,18 @@ ext {
     minSdkVersion = 16
```

Comment:
Should we also update our minSdkVersion to a higher version?

Reply:
I am actually not aware of what is the process for increasing the minSDKVersion. I believe it should be >= minSDK of our consumers. 
minSDKVersion of CP is 26 and that of Auth app is also 26. So, I think we can increase it.

Reply:
minSdkVersion update is a breaking change (For consuming apps)

https://developer.android.com/build/manage-manifests#override_uses-sdk_for_imported_libraries

Reply:
If we want to increase it, then I suggest we increase to 24 which is what the Broker's MIN SDK is. 

Min SDK should be increased with a MAJOR version release of MSAL. MAJOR in the semver world means that client should expect it to be a breaking change.

Reply:
TBH, I would really prefer that we increase it and increase to 24 and mark in changelog as MAJOR. I think this has been long overdue.

Reply:
That said, we may want to run this by @negoe

Reply:
We cannot increase the version to 24 as OneAuth is still on 22. I would want to take this up separately as I do not want to increase many versions in one go. I have created a PBI to track this item. When OneAuth moves to minSDK 24, I will make a change to directly move to 24 https://identitydivision.visualstudio.com/Engineering/_workitems/edit/2667751